templates/abilities.html

```diff
@@ -1,6 +1,16 @@
 {{#each actor.abilities as |item id|}}
 <li class="item flexrow" data-item-id="{{item._id}}" style="color: {{#if data.archived}}grey{{/if}}">
-  <h4 class="item-name"><a class="item-control item-description" title="{{localize 'CYPHERSYSTEM.ShowDescription'}}"><i class="fas fa-angle-{{#if data.showDescription}}down{{else}}right{{/if}} item-angle" aria-hidden="true"></i>{{item.name}}{{#unless (or (eq data.costPoints "0") (eq data.costPoints 0) (eq data.costPoints ""))}} ({{data.costPoints}} {{data.costPool}} point{{#unless (eq data.costPoints "1")}}s{{/unless}}){{/unless}}</a></h4>
+  <h4 class="item-name"><a class="item-control item-description" title="{{localize 'CYPHERSYSTEM.ShowDescription'}}"><i class="fas fa-angle-{{#if data.showDescription}}down{{else}}right{{/if}} item-angle" aria-hidden="true"></i>
+	{{item.name}}{{#unless (or (eq data.costPoints "0") (eq data.costPoints 0) (eq data.costPoints ""))}} ({{data.costPoints}} 
+		{{#if (eq data.costPool "Might")}}
+			{{localize 'CYPHERSYSTEM.Might'}}
+		{{else if (eq data.costPool "Speed")}}
+			{{localize 'CYPHERSYSTEM.Speed'}}
+		{{else}}
+			{{localize 'CYPHERSYSTEM.Intellect'}}
+		{{/if}}
+	){{/unless}}
+  </a></h4>
```

Comment:
Points are missing. Additional if-branch needed, possibly even a different solution. Since German uses compounds so much, we need different strings for all combinations: Might point, Might points, Speed point, Speed points, Intellect point, Intellect points, Pool point, Pool points.

Reply:
I noted why I deleted it:
> Had to do the big "if" statement for abilities.
In order to not make another strange localisation string, I also deleted the "point(s)" part. So now it only shows thing like "Name of ability (2 Might)".
Tell me if you prefer we still investigate other means to do so.

So I will see what can be done.

Reply:
I saw that 😁

I’m just trying to keep it as close as possible to the way the books do it (including how the game terms are capitalized), and in the books it’s never (2 Might), and the language is not to remove 2 Might. It’s always Might points, unless it’s only 1, in which case it’s Might point.

Reply:
Refactored translation